x/proof/types/expected_keepers.go

```diff
@@ -7,21 +7,22 @@ import (
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
+	apptypes "github.com/pokt-network/poktroll/x/application/types"
 	sessiontypes "github.com/pokt-network/poktroll/x/session/types"
 )
 
 type SessionKeeper interface {
 	GetSession(context.Context, *sessiontypes.QueryGetSessionRequest) (*sessiontypes.QueryGetSessionResponse, error)
+	GetBlockHash(ctx context.Context, height int64) []byte
 }
 
 // AccountKeeper defines the expected interface for the Account module.
 type AccountKeeper interface {
-	GetAccount(context.Context, sdk.AccAddress) sdk.AccountI // only used for simulation
-	// Methods imported from account should be defined here
+	GetAccount(context.Context, sdk.AccAddress) sdk.AccountI
 }
 
-// BankKeeper defines the expected interface for the Bank module.
-type BankKeeper interface {
-	SpendableCoins(context.Context, sdk.AccAddress) sdk.Coins
-	// Methods imported from bank should be defined here
```

Comment:
We still need the bank keeper to be available in this package for testing purposes.

Reply:
While it is no longer a dependency of the proof keeper, it's still a dpendency of gateway, application, supplier, and session keepers; all of which we need to construct in the integration tests added in #412 & #417.

Reply:
In fact, the expected keeper interface must also be updated for those PRs: https://github.com/pokt-network/poktroll/pull/412/files#diff-8395873d16465a1ae7a1601264eac2646e93594539f92d123270b5e9c98d7386